static/css/global.css

```diff
@@ -82,53 +82,6 @@ input[type="submit"] {
   border-radius: 3px;
 }
 /*
--------------------------
-    Bootstrap new styles
--------------------------
-*/
-.panel-feature-orange {
-  border-color:#fbae55;
-}
-.panel-feature-orange > .panel-heading {
-  color:#a15a09;
-  background-color:#fbc281;
-  border-color:#fbae55;
-  }
-.panel-feature-orange > .panel-heading + .panel-collapse > .panel-body {
-  border-top-color:#fbae55;
-}
-.panel-feature-orange > .panel-heading .badge {
-  color:#fbc281;
-  background-color:#a15a09;
-}
-.panel-feature-orange > .panel-footer + .panel-collapse > .panel-body {
-  border-bottom-color:#fbae55;
-}
-
-.panel-feature-blue {
-  border-color:#4ca3ce;
-}
-.panel-feature-blue > .panel-heading {
-  color:#084765;
-  background-color:#6fafce;
-  border-color:#4ca3ce;
-  }
-.panel-feature-blue > .panel-heading + .panel-collapse > .panel-body {
-  border-top-color:#4ca3ce;
-}
-.panel-feature-blue > .panel-heading .badge {
-  color:#6fafce;
-  background-color:#084765;
-}
-.panel-feature-blue > .panel-footer + .panel-collapse > .panel-body {
-  border-bottom-color:#4ca3ce;
-}
-.btn-blue {
-  color:#fff;
-  background-color:#19719c;
-  border-color:#2b5D75;;
-}
-/*
 ------------------
```

Comment:
Did you mean to remove all of this? We've lost the styling on the main page (/community/) now.